py_mind_memo/persistence.py

```
@@ -98,3 +98,15 @@ def on_open_template(self, file_path):
                 self.render_callback(root_node=self.model.root)
             except Exception as e:
                 messagebox.showerror("Error", f"Failed to load template from {file_path}: {e}")
+
+    def open_from_path(self, file_path):
+        """指定されたパスからファイルをロードする（編集フラグを変更しない読み込み専用用途向け）。"""
```

Ruff (0.15.9)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)